schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -187,46 +190,224 @@
             "2015-06-15"
           ]
         },
+        "location": {
+          "type": "string"
+        },
         "properties": {
-          "properties": {
-            "publisher": {
-              "type": "string",
-              "minLength": 1,
-              "description": "Microsoft.Compute/extensions - Publisher"
-            },
-            "type": {
-              "type": "string",
-              "minLength": 1,
-              "description": "Microsoft.Compute/extensions - Type"
+          "oneOf": [
```

Comment:
Basically you are enabling three extensions. At the same time, you need to make sure the old ones still work. You may want to do this:
"anyOf": [
  {old ones which take "type" as any string}, 
  {
    "oneOf": [
        { new extension one },
        { new extension two },
        { new extension three }
    ]
  }
]